src/components/calculator/SavedDataSelector.tsx

```diff
@@ -17,11 +18,10 @@ export const SavedDataSelector: React.FunctionComponent<{
   const { t } = useTranslation()
   const prepopulatedOptions = Object.keys(prepopulated)
 
-  let selected: string[] = []
+  let selected2: string[] = []
```

Comment:
why selected**2**?

Reply:
weird scope issues.

Reply:
I'll admit that my enthusiasm for switching it back and debugging is low.

Reply:
well, it at least seems to be weird scope issues with the *next* commit after this one. I'll switch it back for now.